hal/ia32/memory.c

```diff
@@ -29,6 +29,8 @@ extern char __bss_start[], __bss_end[];
 extern char __heap_base[], __heap_limit[];
 extern char __stack_top[], __stack_limit[];
 
+extern void hal_acpiInit(hal_syspage_t *);
```

Comment:
I know it's just one function prototype, but it would be neater to provide a header file for ACPI